### PR TITLE
Neyyi karam dosa

### PR DESCRIPTION
**Pull request by NeelaReddy25**

### CODE REVIEW

File: menu.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 dosa
-karam
-oil
\ No newline at end of file
+extra karam
```
Comment (NeelaReddy25) [inline menu.txt:2]: do we really require extra karam?

Comment (NeelaReddy25) [reply menu.txt:2]: yes if extra karam extra tasty

Comment (NeelaReddy25) [reply menu.txt:2]: Yaa, that's nice

**Review by NeelaReddyidamakanti — APPROVED** [review]
Yes neyyi karam is good

**Pull request merged**